cmake/globalconfig.cmake

```diff
@@ -165,9 +165,6 @@ configure_file(${IVW_CMAKE_TEMPLATES}/inviwocommondefines_template.h
                ${CMAKE_BINARY_DIR}/modules/core/include/inviwo/core/inviwocommondefines.h 
                @ONLY IMMEDIATE)
 
-# Package creation
-option(IVW_PACKAGE_PROJECT "Create Inviwo Package Project" OFF)
```

Comment:
Libraries dependent on our externals (zlib) having install(EXPORT ...) by default produces errors otherwise. Also, I think it is a bit difficult to understand that it is necessary to enable a "packaging" flag to make an installer, but that is not really the resason.

Reply:
ok ic... is there any reason not to also remove the package install option? or does that one require additional dependencies, NSIS etc?

Reply:
I don't think so.